cdap-e2e-tests/src/e2e-test/features/namespaceadmin/NSAdminDesignTime.feature

```diff
@@ -0,0 +1,115 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+@Namespaceadmin
+Feature: NameSpaceAdmin - Validate nameSpace admin design time scenarios
+
+  @Namespaceadmin
+  Scenario:Verify user is able to click the nameSpace admin tab and successfully navigates to the page
+    Given Open Datafusion Project to configure pipeline
+    When Click on the Hamburger bar on the left panel
+    Then Click on NameSpace Admin link from the menu
+    Then Verify that the user is navigated to nameSpace admin page successfully
+
+  @Namespaceadmin
+  Scenario:Validate user is able to open compute profile page and select a provisioner
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on NameSpace Admin link from the menu
+    Then Click on create profile button for "default" Namespace
+    Then Select a provisioner: "existingDataProc" for the compute profile
+    Then Click on close button of compute profile properties page
+
+  @Namespaceadmin
+  Scenario: Validate user is able to create new namespace preferences and able to delete the added namespace preferences successfully
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on NameSpace Admin link from the menu
+    Then Click "preferences" tab from Configuration page for "default" Namespace
+    Then Click on edit namespace preferences to set namespace preferences
+    Then Set nameSpace preferences with key: "keyValue" and value: "nameSpacePreferences1"
+    Then Click on the Save & Close preferences button
+    Then Click on edit namespace preferences to set namespace preferences
+    Then Delete the preferences
+    Then Click on the Save & Close preferences button
+
+  Scenario: Validate user is able to add multiple namespace preferences inside namespace admin successfully
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on NameSpace Admin link from the menu
+    Then Click "preferences" tab from Configuration page for "default" Namespace
+    Then Click on edit namespace preferences to set namespace preferences
+    Then Set nameSpace preferences with key: "keyValue" and value: "nameSpacePreferences2"
+    Then Click on the Save & Close preferences button
+    Then Click on edit namespace preferences to set namespace preferences
+    Then Delete the preferences
```

Comment:
Instead of calling this method 2 times to delete a namespace is it possible to create a step which accepts the name of namespaces as an argument and delete the namespaces by iterating on them ?

Reply:
Yes Vipin , This logic we will try to implement in Phase 3 developement .